gravitee-apim-e2e/api-test/src/applications/application-searching.spec.ts

```diff
@@ -17,7 +17,7 @@ import { afterAll, beforeAll, expect } from '@jest/globals';
 import faker from '@faker-js/faker';
 import { ApplicationsApi } from '@management-apis/ApplicationsApi';
 import { ADMIN_USER, APP_USER, forManagementAsAdminUser, forManagementAsApiUser, forManagementAsAppUser } from '@client-conf/*';
-import { succeed } from '../../lib/jest-utils';
+import { succeed } from '../../../lib/jest-utils';
```

Comment:
maybe we could also have a `@lib` import path for those imports ? (but it's not not really related to your change)